applications/nrf_desktop/src/modules/Kconfig.usb_state

```diff
@@ -57,6 +57,18 @@ config DESKTOP_USB_PM_REQ_NO_PM_LATENCY
 	  latency and ensures high performance. While USB is active, application
 	  power down is blocked anyway.
 
+config DESKTOP_USB_PM_RESTRICT_REMOVE_DELAY_MS
+	int "Delay in milliseconds before removing power manager level restriction"
+	range 0 10000
+	default 3000 if DESKTOP_BLE_ADV_CTRL_SUSPEND_ON_USB
+	default 0
+	depends on DESKTOP_USB_PM_ENABLE
+	help
+	  Delay in milliseconds before removing power manager level restriction.
+	  This is used to allow the USB to be disconnected without immediate system off.
+	  It allows user to take actions, like e.g. restart advertising, after disconnecting
+	  the USB cable without going through reboot.
```

Comment:
Consider also mentioning that:

```
While disconnecting an USB cable, USB suspend might be triggered.
The USB suspend triggers force power down sequence which makes CAF power manager instantly suspend the application.
```

This fact might not be obvious and is quite relevant here